### DISCUSSION TITLE
Extension Sample of Google not working

### DISCUSSION THREAD

**Discussion by xmax11** [community/community#64057] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am trying to create an extension of my website [https://printebill.com/lesco-online-bill/](url) for the google chrome store & I'm facing an issue with the manifest version. can you help
[dawn history override.zip](https://github.com/community/community/files/12367895/dawn.history.override.zip)
 me how to change it from manifest version2 to 3

**Comment by danielkorsh1** [comment]
### yes
would like an update too!! 
hard work.

**Comment by tuves** [comment]
Hi @xmax11,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Chrome Extension Manifest V3](https://developer.chrome.com/docs/extensions/mv3/intro/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
